<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game_Widget</class>
 <widget class="QWidget" name="Game_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/2048bg.jpg);</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="record">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="best_record">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="record_lablel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>得分：</string>
   </property>
  </widget>
  <widget class="QLabel" name="best_record_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最高分</string>
   </property>
  </widget>
  <widget class="QToolButton" name="random_button">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>76</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/random.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>76</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/stop.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hammer_button">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>76</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/hammer.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bar_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/bar.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/2048.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ScriptS</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HAPPY</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_15">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>521</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_13">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>521</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>406</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>291</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_8">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>291</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_10">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>406</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>176</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_14">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>521</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_4">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>176</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>176</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_16">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>521</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>291</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>406</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>176</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_6">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>291</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_12">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>406</y>
     <width>110</width>
     <height>110</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_bg">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>170</y>
     <width>466</width>
     <height>466</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/background.jpg);</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>graphicsView_bg</zorder>
  <zorder>record</zorder>
  <zorder>best_record</zorder>
  <zorder>record_lablel</zorder>
  <zorder>best_record_label</zorder>
  <zorder>random_button</zorder>
  <zorder>stop_button</zorder>
  <zorder>hammer_button</zorder>
  <zorder>bar_button</zorder>
  <zorder>temp</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>graphicsView_15</zorder>
  <zorder>graphicsView_13</zorder>
  <zorder>graphicsView_11</zorder>
  <zorder>graphicsView_7</zorder>
  <zorder>graphicsView_8</zorder>
  <zorder>graphicsView_10</zorder>
  <zorder>graphicsView_1</zorder>
  <zorder>graphicsView_14</zorder>
  <zorder>graphicsView_4</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>graphicsView_16</zorder>
  <zorder>graphicsView_5</zorder>
  <zorder>graphicsView_9</zorder>
  <zorder>graphicsView_3</zorder>
  <zorder>graphicsView_6</zorder>
  <zorder>graphicsView_12</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
